src/archiefvernietigingscomponent/destruction/tests/test_destruction_list_create.py

```diff
@@ -93,3 +93,37 @@ def test_list_with_short_review_process(self):
             b'<script id="short-review-zaaktypes" type="application/json">["http://example.com/zaak/uuid-1"]</script>',
             response.content,
         )
+
+    def test_reminder(self):
+        reviewers = UserFactory.create_batch(2, role__can_review_destruction=True)
+        zaken = [f"http://some.zaken.nl/api/v1/zaken/{i}" for i in range(1, 3)]
+        zaken_identificaties = ["ZAAK-1", "ZAAK-2", "ZAAK-3"]
+
+        url = reverse("destruction:record-manager-create")
+        data = {
+            "name": "test list",
+            "zaken": ",".join(zaken),
+            "reviewer_1": reviewers[0].id,
+            "reviewer_2": reviewers[1].id,
+            "zaken_identificaties": ",".join(zaken_identificaties),
+        }
+
+        response = self.client.post(url, data)
+
+        self.assertRedirects(response, reverse("destruction:record-manager-list"))
+
+        destruction_list = DestructionList.objects.get()
+
+        self.assertEqual(destruction_list.name, "test list")
+        self.assertEqual(destruction_list.author, self.user)
+        self.assertEqual(destruction_list.status, ListStatus.in_progress)
+        self.assertEqual(destruction_list.items.count(), len(zaken))
+
+        response = self.client.get(url, data)
```

Comment:
Here you are calling `self.client.get()` but there are 2 problems:
- The view expects a POST call
- The POST call is already happening on line 111

Then, on line 117-120, you are re-testing for things that are tested by `test_create_list`, so it is not needed.